docs/Homebrew-Governance.md

```diff
@@ -16,131 +17,132 @@
 
 1. New members (unless nominated as maintainers, see below) will be admitted by an ordinary resolution of the PLC and added to the Homebrew organisation on GitHub.
 
-2. Members may vote in all general elections and resolutions, hold office for Homebrew, and participate in all other membership functions.
+2. Members are expected to remain active within Homebrew. Members who are not active maintainers or active committee members must affirm their continued interest in Homebrew membership annually by voting on annual measures, even if voting abstention. Inactive, non-affirmed, non-voting members will be removed within 14 days after the annual meeting unless excused by the PLC.
 
-3. Members are expected to remain active within Homebrew. Members who are not active maintainers or active committee members are required to affirm their continued interest in Homebrew membership annually by voting on annual measures, even if voting abstention. Inactive, unaffirmed, non-voting members will be removed within 14 days after the annual meeting unless excused by the PLC.
+3. A member may be removed from Homebrew by an ordinary resolution of the PLC. A removed member may be reinstated by the usual admission process.
 
-4. A member may be removed from Homebrew by an ordinary resolution of the PLC. A removed member may be reinstated by the usual admission process.
+4. All members will follow the [Homebrew Code of Conduct](https://github.com/Homebrew/.github/blob/HEAD/CODE_OF_CONDUCT.md#code-of-conduct). Changes to the code of conduct must be approved by the PLC.
 
-5. All members will follow the [Homebrew Code of Conduct](https://github.com/Homebrew/.github/blob/HEAD/CODE_OF_CONDUCT.md#code-of-conduct). Changes to the code of conduct must be approved by the PLC.
-
-6. Members should abstain from voting when they have a conflict of interest not shared by other members. No one may be compelled to abstain from voting.
+5. Members should abstain from voting when they have a conflict of interest not shared by other members. No one may be compelled to abstain from voting.
 
 ## 3. General Meetings of Members
 
-1. A general meeting of the members may be called by either an ordinary resolution of the PLC or a majority of the entire membership. The membership must be given at least three weeks notice of a general meeting.
+1. A general meeting of the members may be called by either an ordinary resolution of the PLC or a majority of the entire membership. The membership must be given at least three weeks' notice of a general meeting.
```

Comment:
> or a majority of the entire membership.

Would be good to clarify what "majority of the entire membership" means: 50% of all current members must say yea OR voted yeas > voted nays OR something else?

Reply:
@maxim-belkin This wording has not changed in this PR. Happy to have a conversation about it here but just want to be explicit that this is not considered blocking.